counter_uas_offboard_ctrl_node:
  ros__parameters:
    CUI:
      offboard:
        frameRate: 30
        maxSpeed: 10.0
        maxYawRate: 2.5
        altVelLmt: 0.8

        takeoff:
          altitude: 10.0
          kpGain: 1.0
      
      yawCtrl:
        gain:
          Kp: 0.85
          Ki: 0.01
          Kd: 0.3
          adaptive: 1.0

        filter:
          alpha: 0.2
        
        limits:
          yawAngle: 2.0
          yawRate: 2.5
        
        topic:
          uasState: "mavros/uas_1/state"
          uasOdom: "mavros/uas_1/local_position/odom"
          tgtOdom: "/target_odometry"
          mountAng: "/mavros/uas_1/mount_control/command"
          uasSetPt: "mavros/uas_1/setpoint_raw/local"
        
        service:
          arming: "mavros/uas_1/cmd/arming"
          setMode: "mavros/uas_1/set_mode"